taxi/models.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.db.models import CharField
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> CharField:

```

Comment:
The return type annotation for the `__str__` method should be `str`, not `CharField`. The `__str__` method returns a string representation of the object.